apps/web/src/components/(dashboard)/common/command-menu.tsx

```diff
@@ -113,15 +143,42 @@ export function CommandMenu({ open, onOpenChange }: CommandMenuProps) {
   };
 
   return (
-    <CommandDialog commandProps={{ onKeyDown: handleKeyDown }} open={open} onOpenChange={setOpen}>
+    <CommandDialog
+      commandProps={{
+        onKeyDown: handleKeyDown,
+        filter: (value, search) => {
+          const exists = value.includes(search);
+          if (exists) return 1;
+
+          const result = searchResults.find(
+            (result) => result.label.toLocaleLowerCase() === value.toLocaleLowerCase(),
+          );
+
+          if (result?.recipents.some((recipent) => recipent.includes(search))) return 1;
+          return 0;
+        },
+      }}
```

Comment:
Wonder if we can cheat and just use an `sr-only` thing that contains the recipients so we don't have to add a custom filter fn 🤔

Reply:
not sure what you mean by sr-only? can you please explain?

Reply:
added a new `value` property in searchResults

```
const newResults = searchDocuments?.map((document) => ({
        label: document.title,
        path: `/documents/${document.id}`,
        value: document.title + document.Recipient.map((recipient) => recipient.email).join(' '),
      }));
```

with this we don't need a custom filter function anymore